schedule/functional/rootonly.yaml

```diff
@@ -0,0 +1,34 @@
+name:           rootonly
+description:    >
+    Maintainer: Zaoliang Luo <zluo@suse.com>
+    Installation opensuse-Tumbleweed-DVD-rootonly@64bit, opensuse-welcome excluded
```

Comment:
Please, use the same metadata content as in the original test suite.

```
Maintainer: okurz
Selecting the installer option to skip user creation, i.e. only create the root account. Verifies ssh to ensure login for the single available account is still possible.
Added in https://progress.opensuse.org/issues/47846
```

And since we don't need the "test workaround" anymore, we can simply remove the notice:
```
INFO: Until it is possible to remove the opensuse_welcome module from main.pm/main_common.pm, this test suite needs EXCLUDE_MODULES=opensuse_welcome (See https://progress.opensuse.org/issues/56267 for more information)
```

Reply:
okay, updated.